browser/model/rhamt.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+let fs = require('fs');
+let path = require('path');
+let mkdirp = require('mkdirp');
+let unzip = require('unzip-stream');
+
+
+import InstallableItem from './installable-item';
+import Installer from './helpers/installer';
+import Platform from '../services/platform';
+
+class RhamtInstall extends InstallableItem {
+  constructor(installerDataSvc, targetFolderName, downloadUrl, fileName, sha256sum) {
+    super(RhamtInstall.KEY, downloadUrl, fileName, targetFolderName, installerDataSvc, false);
+    this.addOption('install', this.version, '', true);
+    this.sha256 = sha256sum;
+  }
+
+  static get KEY() {
+    return 'rhamt';
+  }
+
+  installAfterRequirements(progress, success, failure) {
+    progress.setStatus('Installing');
+    let installer = new Installer(this.keyName, progress, success, failure);
+    return new Promise((resolve, reject)=> {
```

Comment:
If simply unzipping is the aim here, why not use what we already have:
https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/browser/model/helpers/installer.js#L78